packages/cli-repl/test/e2e-fle.spec.ts

```diff
@@ -236,6 +236,23 @@ describe('FLE tests', () => {
     expect(plainMongoResult).to.not.include("phoneNumber: '+12874627836445'");
   });
 
+  it('does not allow compactStructuredEncryptionData command when mongo instance configured without auto encryption', async() => {
+    const shell = TestShell.start({
+      args: ['--nodb']
+    });
+    const uri = JSON.stringify(await testServer.connectionString());
+
+    await shell.waitForPrompt();
+
+    await shell.executeLine(`plainMongo = Mongo(${uri});`);
+    await shell.executeLine(`plainMongo.getDB('${dbname}').plain.insertOne({ \
+      phoneNumber: '+12874627836445' \
+    });`);
+
+    const compactResult = await shell.executeLine(`plainMongo.getDB('${dbname}').plain.compactStructuredEncryptionData()`);
+    expect(compactResult).to.include('The "compactStructuredEncryptionData" command requires Mongo instance configured with auto encryption.');
```

Comment:
This can still be simplified quite a bit:

```suggestion
    const shell = TestShell.start({
      args: [await testServer.connectionString()]
    });
    await shell.waitForPrompt();
    
    const compactResult = await shell.executeLine(`db.coll.compactStructuredEncryptionData()`);
    expect(compactResult).to.include('The "compactStructuredEncryptionData" command requires Mongo instance configured with auto encryption.');
```